lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -136,7 +136,12 @@ sub dead {
 sub get_websocket_api_version {
     my ($self) = @_;
 
-    return $self->get_property('WEBSOCKET_API_VERSION');
+    # Cache this value. To avoid keeping querying the DB.
+    unless ($self->{_websocket_api_version_}) {
+        $self->{_websocket_api_version_} = $self->get_property('WEBSOCKET_API_VERSION');
```

Comment:
wasn't it called: INTERFACE_VERSION? I mean the property?. pick one or the other `OpenQA::WebSockets::Server::INTERFACE_VERSION`

Reply:
@foursixnine 

On the DB is WEBSOCKET_API_VERSION. 
This is the distinguinsh from the isotovideo interface.